docs/03-AcceptanceTests.md

```diff
@@ -310,13 +319,65 @@ $I->seeInCurrentUrl('user/1');
 $user_id = $I->grabFromCurrentUrl('~$/user/(\d+)/~');
 ```
 
-## Selenium WebDriver
+## WebDriver: Selenium Server or PhantomJS
 
 A nice feature of Codeception is that most scenarios are similar, no matter of how they are executed.
 `PhpBrowser` was emulating browser requests but how to execute such test in a real browser like Chrome or Firefox? 
 Selenium WebDriver can drive them so in our acceptance tests we can automate scenarios we used to test manually.
 In such tests we should concentrate more on **testing the UI** than on testing functionality.
 
+"[Selenium WebDriver](http://www.seleniumhq.org/projects/webdriver/)" is the name of an API (specified by the Selenium project)
+to drive browsers automatically. Codeception supports two implementations of this API: Selenium Standalone Server and PhantomJS
```

Comment:
To be honest, PhantomJS is deprecated, not maintained, and soon will be overtaken by headless Chrome.
You can connect to Chrome via ChromeDriver natively without Selenium Server, and facebook/webdriver supports this.
This is how Laravel Dusk works. However, people still use Phantom because it is simpler. We keep it in reference but better not to recommend it that much. If it is possible - try to run Codeception with [headless Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome) (didn't have chance to try it yet)